app/code/Magento/Customer/Console/Command/UpgradeHashAlgorithmCommand.php

```diff
@@ -19,31 +19,29 @@
 class UpgradeHashAlgorithmCommand extends Command
 {
     /**
-     * @var CollectionFactory
-     */
-    private $customerCollectionFactory;
-
-    /**
-     * @var Collection
+     * @var Encryptor
      */
-    private $collection;
+    private $encryptor;
 
     /**
-     * @var Encryptor
+     * @var CustomerResourceModel
      */
-    private $encryptor;
+    private $resourceModel;
 
     /**
-     * @param CollectionFactory $customerCollectionFactory
+     * UpgradeHashAlgorithmCommand constructor.
+     *
      * @param Encryptor $encryptor
+     * @param CustomerResourceModel $resourceModel
      */
     public function __construct(
-        CollectionFactory $customerCollectionFactory,
-        Encryptor $encryptor
+        Encryptor $encryptor,
+        CustomerResourceModel $resourceModel
```

Comment:
Hi @atwixfirster,
Could you try to pass Proxy class for the resource model? I think it should resolve the issue.